src/App.tsx

```diff
@@ -2,15 +2,22 @@ import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { Movie } from './types/Movie';
+import { useState } from 'react';
 
 export const App = () => {
+  const [movies, setMovies] = useState(moviesFromServer);
+  const addMovie = (newMovie: Movie) => {
```

Comment:
In the upper component - handle, in the down - on

```suggestion
  const handleAddMovie = (newMovie: Movie) => {
```